diff --git a/node_modules/firebase/firestore/dist/memory/esm/index.js b/node_modules/firebase/firestore/dist/memory/esm/index.js
index f30f146..082b4ac 100644
--- a/node_modules/firebase/firestore/dist/memory/esm/index.js
+++ b/node_modules/firebase/firestore/dist/memory/esm/index.js
@@ -1,4 +1,4 @@
-import { registerVersion } from 'https://www.gstatic.com/firebasejs/10.9.0/firebase-app.js';
+import { registerVersion } from '@firebase/app';
 import { onLog, setLogLevel } from 'https://www.gstatic.com/firebasejs/10.9.0/firebase-util.js';
 import 'https://www.gstatic.com/firebasejs/10.9.0/firebase-auth-interop-types.js';
 import { Component } from 'https://www.gstatic.com/firebasejs/10.9.0/firebase-component.js';
