--- a/node_modules/firebase/dist/esm/index.js
+++ b/node_modules/firebase/dist/esm/index.js
@@ -1,5 +1,5 @@
 import { registerVersion } from 'https://www.gstatic.com/firebasejs/11.9.1/firebase-app.js';
-import { onLog, setLogLevel, getLogLevel, LogLevel } from 'https://www.gstatic.com/firebasejs/11.9.1/firebase-app.js';
+import { onLog, setLogLevel, getLogLevel } from 'https://www.gstatic.com/firebasejs/11.9.1/firebase-app.js';
 export { _isFirebaseApp, _getProvider, _registerComponent, _getLG, _logError, _registerFramework, _getAppCheckToken, _getModularInstance, _addComponent, _addOrOverwriteComponent, _DEFAULT_ENTRY_NAME, _FirebaseService, _createFirebaseNamespace, _extendNamespace, _registerService, _removeServiceInstance } from 'https://www.gstatic.com/firebasejs/11.9.1/firebase-app.js';
 import { initializeApp } from 'https://www.gstatic.com/firebasejs/11.9.1/firebase-app.js';
 export { initializeApp, getApp, getApps, deleteApp, onCLS, onINP, onLCP, onFCP, onTTFB } from 'https://www.gstatic.com/firebasejs/11.9.1/firebase-app.js';
@@ -53,4 +53,4 @@
   registerVersion('app-compat', version);
 }
 
-export { FirebaseError, SDK_VERSION, onLog, setLogLevel, getLogLevel, LogLevel };
+export { FirebaseError, SDK_VERSION, onLog, setLogLevel, getLogLevel };
