diff --git a/node_modules/firebase/node_modules/firebase/dist/index.node.cjs.js b/node_modules/firebase/node_modules/firebase/dist/index.node.cjs.js
index 4016c14..d30779e 100644
--- a/node_modules/firebase/node_modules/firebase/dist/index.node.cjs.js
+++ b/node_modules/firebase/node_modules/firebase/dist/index.node.cjs.js
@@ -107127,7 +107127,7 @@
  * @returns A `Promise` that resolves with the created `Firestore` object.
  */
 function initializeFirestore(app, settings) {
-    const provider = new modular_exp.FirestoreProvider();
+    const provider = new modular_exp.FirestoreProvider('node-cjs');
     const firestore = provider.getImmediate({
         app,
         settings
diff --git a/node_modules/firebase/node_modules/firebase/dist/index.node.mjs b/node_modules/firebase/node_modules/firebase/dist/index.node.mjs
index 548325e..e70d649 100644
--- a/node_modules/firebase/node_modules/firebase/dist/index.node.mjs
+++ b/node_modules/firebase/node_modules/firebase/dist/index.node.mjs
@@ -107085,7 +107085,7 @@
  * @returns A `Promise` that resolves with the created `Firestore` object.
  */
 function initializeFirestore(app, settings) {
-    const provider = new FirestoreProvider();
+    const provider = new FirestoreProvider('node-esm');
     const firestore = provider.getImmediate({
         app,
         settings